staging/src/k8s.io/sample-apiserver/README.md

```diff
@@ -2,6 +2,7 @@
 
 Demonstration of how to use the k8s.io/apiserver library to build a functional API server.
 
+**Note:** go-get or vendor this package as `k8s.io/sample-apiserver`.
```

Comment:
not sure how this fixes the sample-controller issues? isn't this for sample-apiserver?

Reply:
oops. Yes :) But, we should add that to all staging repos, if we don't say that already.

Reply:
> But, we should add that to all staging repos, if we don't say that already.

:+1:

Reply:
Added a note to all repos (other than client-go which has the README outside of staging).